student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative('./person')
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(*args, classroom: 'Unknown', **kwargs)
+    super(*args, **kwargs)
+    @classroom = classroom
+  end
```

Comment:
- [ ] In the `initialize` method argument, the value `"unknown"` should be assigned to `name` and not to the classroom variable.

Reply:
My bad thanks for that 😃.